docs/howto/create-a-workspace.md

```diff
@@ -0,0 +1,28 @@
+# Creating a Workspace
+A Workspace is a directory containing information about one or more SDK produced plugins to be packaged into a single, deployable `.yxi` archive. A given Workspace contains a `ayx_workspace.json`, and all other necessary components to build plugins.
+
+To create a Workspace, we first create a new, empty directory. Then, we run the sdk-workspace-init command inside that directory and fill out the prompts, which starts the workspace initialization process.

```

Comment:
TW review:

To create a Workspace, we first create a new, empty directory. Then, we run the `sdk-workspace-init` command inside that directory and fill out the prompts, which starts the workspace initialization process.